LumbApp/Conectores/ConectorSI/ConectorSI.cs

```diff
@@ -74,7 +74,32 @@ private string DetectarArduinoPort()
         }
 
         public void ActivarSensado() { sensando = true; }
-        public void Desconectar() { mySerialPort.Close(); }
+        public void Desconectar() {
+            
+            string datos;
+            try
+            {
+                for (int i = 0; i < 50; i++)
+                {
+
+                    mySerialPort.WriteLine("a");
+
+                    datos = mySerialPort.ReadTo("$");
+
+                    if (datos.Contains("88"))
+                    {
+                        break;
+                    }
```

Comment:
pregunta, que hace este for? XD

Reply:
Le envía al arduino la señal de apagado hasta que este le contesté con 88 que significa que lo recibió bien. Lo hacemos muchas veces por las dudas porque a veces no llega bien el msj

Reply:
"a" es apagar? jaja
OK, me parece bien, pero porfas dejemos comentarios y documentacion de la funcion